test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -125,19 +121,6 @@ protected override void Cleanup()
 
         protected bool IsPassing { get { return _passed; } }
 
-        // Return the location of the local Samples folder
-        // Used primarily for local Scope runs
-        protected string SamplesDir { get { return Path.Combine(RootDir, SamplesRootRelPath); } }
-
-        // Return the location of the local scope folder under Samples. Used primarily 
-        // by Scope scripts and for Scope tests
-        protected string ScopeSamplesDir { get { return Path.Combine(RootDir, ScopeRootRelPath); } }
-
-        // Return the local of the directory where external binaries for test purposes are located
-        protected string ExternalTestBinariesDir { get { return Path.Combine(RootDir, TestExtDir); } }
-
-        protected string TestDirectory { get { return Path.Combine(RootDir, TestDir); } }
-
```

Comment:
cc @yaeld to check whether this conflicts with her work.

Reply:
Thanks for thinking of that work, @sfilipi.

If there are things that are only used in the internal repo, we should move them to only be in the internal repo. It isn't sustainable to have a bunch of methods in dotnet/machinelearning that are not used.

@yaeldekel - if any of these are necessary for your work, can you add them to a `partial` class in the internal repo?